### PR TITLE
Fix merge conflicts again.

### PR DESCRIPTION
**Pull request by jason-fox**
Merge branch 'master' into feature/842_ngsi_ld - fix conflicts again.

### CODE REVIEW

File: lib/services/devices/deviceRegistryMongoDB.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const errors = require('../../errors');
 const constants = require('../../constants');
 const Device = require('../../model/Device');
 const async = require('async');
-const context = {
+let context = {
```
Comment (jason-fox) [inline lib/services/devices/deviceRegistryMongoDB.js:33]: This looks very strange - it is necessary from #901  - it looks like that PR is amending this constant - is this correct?

Comment (fgalan) [reply lib/services/devices/deviceRegistryMongoDB.js:33]: Not sure... maybe @AlvaroVega could provide feedback on this.

Comment (AlvaroVega) [reply lib/services/devices/deviceRegistryMongoDB.js:33]: I guess there is no problem no move from `const` to `let`

Comment (fgalan) [reply lib/services/devices/deviceRegistryMongoDB.js:33]: So I understand it's correct and NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**